frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadList/uploadListItem/components/uploadListItemRevisionTag/uploadListItemRevisionTag.styles.ts

```diff
@@ -28,6 +28,12 @@ export const RevisionTagField = styled(FormTextFieldBase)<{ $selectedrow: boolea
 	.MuiOutlinedInput-root {
 		height: 100%;
 		border-radius: 8px;
+		${({ theme, $selected }) => $selected && css`
+			background-color: ${theme.palette.secondary.light};
+			input {
+				color: ${theme.palette.primary.contrast};
```

Comment:
I am not sure whether it is because of this code, but not, when a input from a selected row is focused AND hovered, the border loses the color:
**Focused NOT hovered**
![image](https://github.com/3drepo/3drepo.io/assets/43547703/d22bf141-0e1a-42fa-8cc4-bff63a6f510c)

**Focused AND hovered**
![image](https://github.com/3drepo/3drepo.io/assets/43547703/824aab35-2fba-4015-9b10-5d6cb4ec1c4f)